spec/controllers/account/dashboard_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Account::DashboardController, type: :controller do
+  describe "GET #index" do
+    it "renders the :index view" do
+      get :index
+      expect(response).not_to have_http_status(200)
+      expect(response).to have_http_status(302)
```

Comment:
so either first expectation or second, both are not necessary

Reply:
Thanks, fixed.